acquire/acquire.py

```diff
@@ -404,11 +404,15 @@ def get_spec_additions(cls, target):
 @local_module
 class WinRDPSessions(Module):
     DESC = "Windows Remote Desktop session information"
-    SPEC = [
-        ("command", (["qwinsta", "/VM"], "win-rd-sessions")),
-    ]
     EXEC_ORDER = ExecutionOrder.BOTTOM
 
+    @classmethod
+    def get_spec_additions(cls, target):
+        qwinsta = subprocess.run(["where", "qwinsta.exe"], capture_output=True, text=True).stdout.rstrip("\n")
+        return [
+            ("command", ([qwinsta, "/VM"], "win-rdp-sessions")),
```

Comment:
Just a question, was it also a bug that the previous command did `win-rd-sessions` instead of `win-rdp-sessions`?

Reply:
Not intentional but I keep typing 'rdp' instead of 'rd' and even Google corrects my search automatically (just a test). So I think we better change this yeah.

Reply:
But I assume it's functionally the same?

Reply:
Yes functionally the same, but it the output of the command will now be written to `win-rdp-sessions` instead of `win-rd-sessions`. I guess that's fine, as the module is also called `WinRDPSessions` and there is not yet a plugin on the dissect.target side that consumes this file.